src/devices_models/devices/common/activepower_constraints.jl

```diff
@@ -0,0 +1,116 @@
+
+struct ActivePowerConstraintsInputs
+    limits::Function
+    range_constraint::Function
+    multiplier::Function
+    timeseries_func::Function
+    timeseries_func_kwargs::Dict{Symbol, Any}
```

Comment:
These names could be better.   Make suggestions if you have them.

Reply:
I don't think we pass kwargs here anymore. Any kwargs calls outside of the public API functions is stale code that hasn't been updated.

Reply:
I'm using this for `M_value` which is used in `device_timeseries_ub_bigM`

Reply:
Let's define it as a constant in Definitions and not pass it anymore.